packit/cli/propose_downstream.py

```diff
@@ -86,6 +86,13 @@ def get_dg_branches(api, dist_git_branch):
     is_flag=True,
     help="Don't discard changes in the git repo by default, unless this is set.",
 )
+@click.option(
+    "--use-downstream-specfile",
+    default=False,
+    is_flag=True,
+    help="Perform a pull-from-upstream. The same as a propose-downstream but "
+    "you don't need having the packit config file or the specfile in the upstream repo.",
```

Comment:
```suggestion
    help="Do not require having the Packit config file or the specfile in the upstream repo.",
```
not sure whether I would mention here the `pull-from-upstream` directly as that is service specific :thinking: 

I was also thinking whether it would make sense to introduce a "new" command `packit pull-from-upstream` that would basically do the same using this option, but not sure whether that would bring some more clearness to users.

Reply:
I was thinking the same. I think it is far more clear if we could have a separate command. I will try to create it.

Reply:
+1

We've been already asked multiple times about `packit pull-from-upstream` CLI commands so it's probably what people expect to exist.